reframe/utility/__init__.py

```diff
@@ -41,9 +41,13 @@ def import_module_from_file(filename):
     """Import module from file."""
 
     filename = os.path.normpath(os.path.expandvars(filename))
+
     if os.path.isdir(filename):
         filename = os.path.join(filename, '__init__.py')
 
+    if filename.startswith('..'):
+        filename = os.path.abspath(filename)
```

Comment:
I think that this is the only solution. It seems we cannot use the `importlib.import_module()`, because we don't which package to attach to. Am I right?

Reply:
yes, that is correct.

Reply:
I tested and it now works both with settings and checks defined with relative paths.